Update installation and running first file.md

deonchan2012 opened the pull request:
Initial draft of getting started page.

deonchan2012 committed:
Update installation and running first file.md

initial commit

YahyaElgabra commented:
I've reviewed the recent changes to the Installation and Getting Started documentation for Saturn. The instructions are clearly outlined, and the step-by-step guidance for different operating systems is particularly helpful for new users.

However, there are some areas where we could enhance the documentation further:

Inclusion of Images: The current documentation would greatly benefit from screenshots. For example, they can make the installation process more approachable, especially for users who are not as familiar with installing software from GitHub releases.

Misleading test: In one part, you meantion that "For general use, download the file ending with setup.exe.", which could be misleading as Linux users, for instance, can't run .exe files

Bad Indentation: It seems like the indentation of your example code under the gettign started section is broken, please work on it such that it follows clean standards

Better explanation: It might be useful to describe what the .data and .text sections are, since a newcomer might not be fmailiar with them

deonchan2012 commented:
I will get on it!

 I agree with your thoughts on images and will add them.

For the misleading text I actually did not know, thank you for reading through that area.

I will also explain the example code better.

deonchan2012 committed:
Update installation and running first file.md

Update example code to look more clean as well as move .exe file to windows section

deonchan2012 committed:
Update installation and running first file.md

added iamges

krishnacheemalapati commented:
Great work!
